src/pages/gateway/cache-control-headers.md

```diff
@@ -13,11 +13,11 @@ keywords:
 
 # Edge caching
 
-API Mesh for Adobe Developer App Builder supports edge caching if you provide your own content delivery network (CDN), such as Fastly. Edge caching helps improve website load times and reduces consumption costs associated with bandwidth.
+API Mesh for Adobe Developer App Builder supports edge caching if you provide your own content delivery network (CDN), such as [Fastly](./fastly.md). Edge caching helps improve website load times and reduces consumption costs associated with bandwidth.
 
 <InlineAlert variant="info" slots="text"/>
 
-When using a CDN, you must invalidate the cache after modifying a mesh configuration or you will receive stale information.
+When using a CDN, you must invalidate the cache after modifying a mesh configuration, or you will receive stale information.
```

Comment:
is Edge caching = CDN here then or what other types of edge caching did we refer to?
see how this is ambiguous?